.github/workflows/linter.yml

```diff
@@ -0,0 +1,25 @@
+name: Lint Code Base
+
+# Documentation:
+# https://help.github.com/en/articles/workflow-syntax-for-github-actions
+
+on: pull_request
+
+jobs:
+  build:
+    name: Lint Code Base
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Lint Code Base
+        uses: github/super-linter@v3
+        env:
+          VALIDATE_ALL_CODEBASE: false
+          VALIDATE_BASH: false
+          VALIDATE_PYTHON: false
+          VALIDATE_PYTHON_FLAKE8: false
+          VALIDATE_PYTHON_BLACK: false
+          VALIDATE_YAML: false
```

Comment:
Isn't `VALIDATE_YAML` the only one we need to disallow (because gotemplate in helm "yaml" files will never validate)? Just curious why we need to bother disallowing the others

Reply:
Unfortunately no. There is a bash script and some python code where the linter complains as well.

Reply:
That's fine. This seems worth it. If in future it becomes a pain, someone can always make a GH Action for DavidAnson/markdownlint only.